### ISSUE TITLE
Ringing sound cannot be disabled/muted (VoIP, voice/video call, notifications)

### ISSUE THREAD

**Issue by fooness** [vector-im/element-web#15591]
### Description

As described in https://github.com/vector-im/element-web/issues/5031 and https://github.com/vector-im/element-web/issues/5891, the ringtone is … noisy. 🙉

And unfortunately it seems the call notification cannot be disabled.
There’s two settings in the `Notifications` tab:

Enable audible notifications for this session `OFF [x    ] ON`
Call invitation `[x] OFF` `[  ] ON` `[  ] NOISY`

After setting both options to OFF, the ringtone still _shouts at me_.
Clearing the cache and reloading also did not help.

### Version information

- **Browser**: Firefox 82.0, Chromium 88.0.4290.0
- **OS**: macOS 10.14.6
- **URL**: 1.7.11 and prior

**Comment by fooness** [comment]
This still affects 1.7.12 and 1.7.13 unfortunately.

**Comment by ajayyy** [comment]
Is there a reason this is not considered a bug? If this is not intended, then the setting "Enable audible notifications for this session" should be renamed, as it implies that ringtones should be disabled as well.

**Comment by jryans** [comment]
Sure, on second thought, it's a bit closer to a bug... AFAIK, mainly we've just never had a setting that controls this sound, despite the naming of the options, so that's why I went for suggestion at first.

**Comment by fooness** [comment]
I cannot quite understand how this is not considered a major problem. It is no option to rely on browsers being able to mute specific tabs. Please let’s fix this high severity issue which is causing lots of trouble. 🤞

**Comment by Flam3z** [comment]
I hope this gets adressed soon as the noise truly hurts ny ears and that shouldn't be a trade off for users to call their friends.

**Comment by gerroon** [comment]
I can confirm this on Debian Testing Xfce. I cannot disable it, and it is very loud and PulseAudio thinks it is Chromium, when I disabled it fully in PulseAudio for Element, then I can't get sound in Chromium :(

**Comment by ajayyy** [comment]
I have added a fake third user to my calls to use jitsi instead of 1:1 calls to attempt to avoid this issue
